### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by ikoote1**
#POPUP WINDOW.
- Designed a popup window for mobile and desktop using HTML.
- Styled my popup windows in CSS.
- Added appear and disappear using on-click.
- Improved the response with javascript.

### CODE REVIEW

**Review by CollinsTatang — COMMENTED** [review]
Hi @ikoote1, 

While you made a great effort in this project. Unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have all the README file details.

- [x] Please Update your README file following the [README template](https://github.com/microverseinc/readme-template).
- [x] Please make sure that you add the section **Future Features**, including 1-3 potential features to be added.
- [x] Please remove the Microverse logo from your README file

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,289 +1,318 @@
...
+                    <h2 class="proff">Data Dashboard Healthcare</h2>
+                    <p class="daily1">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required. has been the industry's standard</p>
+                    <ul class="languagess">
+                        <li class="g-item1">html</li>
+                        <li class="g-item1">css</li>
+                        <li class="g-item1">bootstrap</li>
+                        <li class="g-item1">ruby</li>
+                    </ul>
+                    <button type="button" onclick="showModal(3);" class="but">See project</button>
```
Comment (Olamarx) [inline index.html]: - [x] I would advise you to follow the principle of separation of concerns. I mean you can use the addEventListener provided by Javascript.

File: index.js [hunk 2]
```diff
@@ -11,3 +12,111 @@ x.addEventListener('click', () => {
...
+
+// Show Modal
+const showModal = (i) => {
+  modals.classList.toggle('active-modal');
+  document.querySelector('.pop-BG').style.display = 'block';
+  document.querySelector(`.modal-${i}`).style.display = 'block';
+};
+
+// Close Modal
+function closeModal(i) {
```
Comment (Olamarx) [inline index.js]: - [x] I would recommend you use the ES6 arrow function as you did for showModal. This is for best practice and shows you are up to date in the industry.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @ikoote1,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- popup functionality ✔️ 

## Required Changes ♻️
_Check the comments under the review._
- [x] Kindly watch this [video](https://www.loom.com/share/8742d809141d4ed5a20439d9ef80c5a3) where I explained somethings.
- [x] I would advise you to remove the whole work section and add it dynamically with javascript just as you did for the modal where you have the data in an array and you iterate over it. This is one of the requirements.
- [x] Once more, kindly make sure the pop-up follows the proposed design in Figma(font, images, text, margins). The font size for the project name on both mobile and desktop is not the same as in Figma. Please do some adjustments.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 3]
```diff
@@ -119,6 +119,120 @@ function closeModal(i) {
...
+</div>
+ 
+`;
+});
+
+// WORK SECTION 2
+const modalsO = [
+
+  {
+    modalTitle: 'Data Dashboard Healthcare',
```
Comment (franclobo) [inline index.js:174]: - **[OPTIONAL]** Please 🙏🏽 review that the title on the modal page corresponds to the card selected, Please fix the titles n the modal pages for a better user interface.

File: index.js [hunk 4]
```diff
@@ -119,6 +119,120 @@ function closeModal(i) {
...
+// work section 1.
+
+const modalsOb = [
+  {
+    modalTitl: 'Components website',
+    modalLis: ['css', 'html', 'Bootstrap', 'Ruby'],
+    modalIm: './images/Img Placeholder.png',
+    modalTex:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+  }];
```
Comment (franclobo) [inline index.js:131]: - **[OPTIONAL]** Kindly correct the pop-up window position for a better user interface, align the modal window to the center and use `overflow-y: scroll;` if the height exceeded the screen size.

| Figma design | Your design |
|---|---|
| image | image |

- **[OPTIONAL]** Please 🙏🏽 avoid the horizontal scroll on the window.

**Review by franclobo — APPROVED** [review]
Hi @ikoote1, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**